service/src/main/kotlin/fi/espoo/evaka/daycare/DaycareQueries.kt

```diff
@@ -418,12 +418,13 @@ WHERE daycare_id = ${bind(daycareId)}
         }
         .toList()
 
-fun Database.Read.getUnitFeatures(): List<UnitFeatures> =
+fun Database.Read.getUnitFeatures(today: LocalDate): List<UnitFeatures> =
     createQuery {
             sql(
                 """
 SELECT id, name, enabled_pilot_features AS features, provider_type, type
 FROM daycare
+WHERE closing_date IS NULL OR closing_date > ${bind(today)}
```

Comment:
Nitpicking - shows closed units that are closed today

Reply:
Hmm I don't even know if that means that the unit is still open today or not. Usually evaka's end dates are inclusive so the unit would still be open on the closing date, but not sure.